countNumberOfLowercase.js

```diff
@@ -0,0 +1,13 @@
+export const typeOfArgumentErrorMessage = "There should be a string!"
```

Comment:
name of this variable tells me `what shouldn't be` but doesn't tell me `what should be`.
Consider renaming to something like `notStringTypeErrorMessage` or similar

Reply:
done